src/pages/Policy.jsx

```diff
@@ -1,9 +1,160 @@
-import React from 'react'
+import React from "react";
+import image1 from "../Images/privacy (1).png";
```

Comment:
all static image files should be exported separately from an index file and finally destructure and import specific files wherever required